### PR TITLE
about me text box

### PR DESCRIPTION
**Pull request by josejimenezolivas**
Alt Text

### CODE REVIEW

File: src/main/java/com/google/codeu/servlets/AboutMeServlet.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  response.setContentType("text/html");
+  
+  String user = request.getParameter("user");
+  
+  if(user == null || user.equals("")) {
+   // Request is invalid, return empty response
+   return;
+  }
+  
+  String aboutMe = "This is " + user + "'s about me.";
```
Comment (elithrar) [inline src/main/java/com/google/codeu/servlets/AboutMeServlet.java]: @jimenezjose - I would look at using `String.format` here - see https://dzone.com/articles/java-string-format-examples

e.g. `String aboutMe = String.format("This is %s's About Me", user);`

**Review by elithrar — DISMISSED** [review]
Looks good - note the minor comment. I know that it wasn't in the guide, but good practice / much easier than concatenating strings with quotes!

Will leave it up to you in this case to change it / leave it.

**Comment by michelaburns** [comment]
Seeing that Matt approved the changes, it's probably good. In the future please include a description!

**Comment by josejimenezolivas** [comment]
I created a new data storage class: User, for the AboutMe section.

**Review by josejimenezolivas — COMMENTED** [review]
Mostly changed the data storage for the user. Hence for the existence of User.java

**Comment by elithrar** [comment]
Hey @jimenezjose - can you resolve the conflict here and merge it? Shouldn't take too long. Reach out on chat or via email if you get stuck - both I & the others can help!

**Review by michelaburns — APPROVED** [review]
Hey! This looks good! Please make sure to merge this and also send out pull requests for your guided projects :)

**Pull request merged**